plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -7445,7 +7443,7 @@ private Answer execute(PrepareUnmanageVMInstanceCommand cmd) {
                 mgr.prepareSecondaryStorageStore(secStoreUrl, secStoreId);
                 ManagedObjectReference morSecDs = prepareSecondaryDatastoreOnSpecificHost(secStoreUrl, targetHyperHost);
                 if (morSecDs == null) {
-                    String msg = "Failed to prepare secondary storage on host, secondary store url: " + secStoreUrl;
+                    String msg = String.format("Failed to prepare secondary storage on host, secondary store url: %s", secStoreUrl);
```

Comment:
As the var is used only once, we could implement this message directly in the `throw new ...`.

Reply:
Done